core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreaker.java

```diff
@@ -73,4 +73,10 @@ static CircuitBreaker ofDefaultName() {
      * Decides whether a request should be sent or failed depending on the current circuit state.
      */
     boolean canRequest();
+
+    /**
+     * Transitions to the specified {@link CircuitState}.
+     * Note that this method will always re-initialize the internal state.
```

Comment:
Could you rephrase this so that it is more useful to the users? What actually happens to our users' experience?